dev-requirements.txt

```diff
@@ -19,3 +19,9 @@ black>=22.8,<22.9
 setuptools>56
 # Debuggery
 icecream>=2.1
+# typing
+mypy>=0.942
+mypy-extensions>=0.4.3
+typed-ast>=1.4.3
+types-mock>=0.1.3
+types-PyYAML>=5.4.3
```

Comment:
I don't think any of these are relevant (e.g. mypy-extensions appears to be a dead project for experimental features, types-mock provides types for a library not installed or imported anywhere, etc.).

Reply:
This is actually cruft from a previous rebase. The `mypy-extensions` should be `typing-extensions` and `types-mock` isn't needed because types-mock was removed.

Reply:
Cool, if you add back in typing-extensions, it generally seem good practice to do it conditionally on the Python version (e.g. if all features you need are in Python 3.8+, then only install it on python_version<3.8).

Reply:
Yeah, I'd agree with you but I'm sure @bitprophet will want to vender it. 

My goal is to later add some additional security with LiteralSring. https://peps.python.org/pep-0675/